parm/config/config.resources

```diff
@@ -191,6 +192,80 @@ elif [[ "${step}" = "atmanalpost" ]]; then
     export npe_node_atmanalpost
     export is_exclusive=True
 
+elif [[ "${step}" = "landanlinit" ]]; then
+
+   # below lines are for creating JEDI YAML
+   case ${CASE} in
+     C768)
+        layout_x=6
+        layout_y=6
+        ;;
+      C384)
+        layout_x=5
+        layout_y=5
+        ;;
+     C192 | C96 | C48)
+        layout_x=1
+        layout_y=1
+        ;;
+      *)
+          echo "FATAL ERROR: Resolution not supported for land analysis'"
+          exit 1
+    esac
+
+    export layout_x
+    export layout_y
+
+    export wtime_landanlinit="00:10:00"
+    export npe_landanlinit=1
+    export nth_landanlinit=1
+    npe_node_landanlinit=$(echo "${npe_node_max} / ${nth_landanlinit}" | bc)
+    export npe_node_landanlinit
+    export memory_landanlinit="3072M"
+
+elif [[ "${step}" = "landanlrun" ]]; then
+
+   case ${CASE} in
+     C768)
+        layout_x=6
+        layout_y=6
+        ;;
+      C384)
+        layout_x=5
+        layout_y=5
+        ;;
+     C192 | C96 | C48)
+        layout_x=1
+        layout_y=1
+        ;;
+      *)
+          echo "FATAL: Resolution not supported'"
+          exit 1
+    esac
+
+    export layout_x
+    export layout_y
+
+    export wtime_landanlrun="00:30:00"
+    npe_landanlrun=$(echo "${layout_x} * ${layout_y} * 6" | bc)
+    export npe_landanlrun
+    npe_landanlrun_gfs=$(echo "${layout_x} * ${layout_y} * 6" | bc)
+    export npe_landanlrun_gfs
+    export nth_landanlrun=1
+    export nth_landanlrun_gfs=1
+    npe_node_landanlrun=$(echo "${npe_node_max} / ${nth_landanlrun}" | bc)
+    export npe_node_landanlrun
+    export is_exclusive=True
+
+elif [[ "${step}" = "landanlfinal" ]]; then
+
+    export wtime_landanlfinal="00:10:00"
+    export npe_landanlfinal=1
+    export nth_landanlfinal=1
+    npe_node_landanlfinal=$(echo "${npe_node_max} / ${nth_landanlfinal}" | bc)
+    export npe_node_landanlfinal
+    export memory_landanlfinal="3072M"
```

Comment:
`landanlinit` and `landanlfinal` are largely the same. (`layout` are not needed for `final` but does that matter if they are?)
Can we condense them in single block?

Reply:
The `landanlinit` and `landanlfinal` sections were added by following `aeroanlinit` and `aeroanlfinal` sections. @CoryMartin-NOAA Would you please help me to answer the @aerorahul questions? Thanks.

Reply:
@aerorahul doesn't the XML generator key off of things like `wtime_$JOBNAME`? I think that is the reason why I had them separate but identical.

Reply:
@WalterKolczynski-NOAA Would you please show me your example in your mentioned fcst/efcs section so that I can make the corresponding changes? Thanks.

Reply:
https://github.com/NOAA-EMC/global-workflow/blob/develop/parm/config/config.resources#L502

Reply:
@WalterKolczynski-NOAA I tried to make some changes, but these changes caused some SHELLCHECK_WARNING in other code as below:

![image](https://user-images.githubusercontent.com/27369703/224517947-a7092b40-02df-4ea3-bf57-f420e0a532f4.png)

How do I fix these issues? Any suggestions? Thanks.

Reply:
For any shellcheck code, you can look up the error code in the shellcheck wiki to see what it's about and example fixes: https://www.shellcheck.net/wiki/SC2086

This one though is on a line you didn't actually change, so just ignore it.

Reply:
Thanks @WalterKolczynski-NOAA for your information.

Reply:
@WalterKolczynski-NOAA I have used the suggested `declare -x` to combine `landanlinit` and `landanlfinal` into a single block. Thanks.